modin/pandas/base.py

```diff
@@ -122,6 +122,23 @@ class BasePandasDataset(ClassLogger):
     # but lives in "pandas" namespace.
     _pandas_class = pandas.core.generic.NDFrame
 
+    def _is_dataframe(self) -> bool:
```

Comment:
Wouldn't it be better to do so?
```suggestion
    # For BasePandasDataset
    _has_columns = None
    # For DataFrame
    _has_columns = True
    # For Series
    _has_columns = False
```

I just don't really like the approach when we need to use the import of high-level objects in low-level ones.

Reply:
or even have a `._typ` field for more verbosity. Pandas do so to mark [`dataframes`](https://github.com/pandas-dev/pandas/blob/8dab54d6573f7186ff0c3b6364d5e4dd635ff3e7/pandas/core/frame.py#L593) and [`series`](https://github.com/pandas-dev/pandas/blob/8dab54d6573f7186ff0c3b6364d5e4dd635ff3e7/pandas/core/series.py#L314) objects

Reply:
I accepted @vnlitvinov's suggestion of using `_pandas_class`, which already exists.